src/BenchmarkDotNet/Loggers/ConsoleLogger.cs

```diff
@@ -26,6 +39,9 @@ public sealed class ConsoleLogger : ILogger
 
         private void Write(LogKind logKind, Action<string> write, string text)
         {
+            if (!unicodeSupport)
+                text = text.ToAscii();
```

Comment:
I wonder what kind of performance impact this change is going to have. We are by default calling `string.Replace("μ", "u")` for every string that is logged to the console

Reply:
I don't expect a noticeable performance impact because:
1. Typically, we perform a lot of heavy string concatenation in logging.
2. `Console.Write` itself takes a lot of time (much more than string.Replace of a single char).
3. We don't have too much logs and I don't think that we should care about such things.